modules/aidemBidAdapter.js

```diff
@@ -149,7 +151,11 @@ function getPageUrl(bidderRequest) {
 }
 
 function buildWinNotice(bid) {
+  const params = bid.params[0]
```

Comment:
there's a bunch of missing semicolons in this file, please fix

Reply:
Done